client/jetbrains/src/main/java/com/sourcegraph/config/PluginSettingChangeContext.java

```diff
@@ -11,6 +11,7 @@ public class PluginSettingChangeContext {
   public final boolean oldCodyVerboseDebugEnabled;
 
   @NotNull public final String newUrl;
+  public final Integer oldCustomAutocompleteColor;
```

Comment:
can this be a `null`?
```suggestion
  public final @Nullable Integer oldCustomAutocompleteColor;
```